module/applications/actor/character-generation-prompt.mjs

```diff
@@ -252,7 +282,7 @@ export default class CharacterGenerationPrompt extends HandlebarsApplicationMixi
     // Spells
     context.availableSpellPoints = await this.charGenData.getAvailableSpellPoints();
     context.maxSpellPoints = await this.charGenData.getMaxSpellPoints();
-    context.spells = this.spells.filter( spell => spell.system.magicType === this.magicType );
+    context.spells = this.spells.filter( spell => spell.system.spellcastingType === this.magicType );
```

Comment:
For consistency, I would probably rename the `magicType` field in this class to `spellcastingType` as well?

Reply:
I thought about it, but I really like that it is something different. it makes it easier to understand for me. If this is enough for your inner Monk I would like to keep it htat way